### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by MaksimkaSota**
1. Added a project deployment logic (firebase.google.com);
2. Updated the README file with additional information;
3. Fixed some remote bugs.

### CODE REVIEW

File: src/utils/languageLocalization/contentText.ts [hunk 1]
```diff
@@ -212,6 +208,18 @@ export const contentText: LanguageLocalizationType = {
     en: 'Dark',
     ru: 'Темная',
   },
+  hintText: {
```
Comment (ZhekaVasil) [inline src/utils/languageLocalization/contentText.ts]: hintText not best option for naming =)
Let's make it more specific - copyHintText or smth like that
hintText sounds more general, you may have another hints in future and the naming may be confusing

File: src/utils/languageLocalization/validationText.ts [hunk 2]
```diff
@@ -25,36 +17,33 @@ export const validationText: LanguageLocalizationType = {
...
+    ru: 'Не более 30 символов',
+  },
+  max100: {
+    en: 'Must be not more than 100 symbols',
+    ru: 'Не более 100 символов',
+  },
+  max200: {
+    en: 'Must be not more than 200 symbols',
+    ru: 'Не более 200 символов',
   },
```
Comment (ZhekaVasil) [inline src/utils/languageLocalization/validationText.ts]: I would propose to make it more configurable =)
I could propose to create new util like
```
export const stringReplacer = (string, params, flag = 'i') => {
  let result = string;
  for (const [key, value] of Object.entries(params)) {
    const regexp = new RegExp(`${key}`, flag);
    result = result.replace(regexp, value);
  }
  return result;
};
```
And use it like:
```
 maxTerm: {
    en: 'Must be not more than %{number} symbols',
    ru: 'Должно быть не более %{number} символов',
    }

stringReplacer(validationText.maxTerm[languageMode], { '%{number}': 30})
stringReplacer(validationText.maxTerm[languageMode], { '%{number}': 50})
```
As result you will have only one string with dynamic param

File: src/utils/languageLocalization/contentText.ts (hunk 1 above)
Comment (MaksimkaSota) [reply src/utils/languageLocalization/contentText.ts]: Done!

File: src/utils/languageLocalization/validationText.ts (hunk 2 above)
Comment (MaksimkaSota) [reply src/utils/languageLocalization/validationText.ts]: Done!

**Review by ZhekaVasil — APPROVED** [review]

**Pull request merged**